src/components/stories/index.js

```diff
@@ -11,9 +13,54 @@ const Stories = () => {
   const privateStories = stories.filter((s) => s.visibility === "PRIVATE");
   const publicStories = stories.filter((s) => s.visibility === "PUBLIC");
 
+  const [image, setImage] = useState(null);
+  const [story, setStory] = useState(null);
+
+  const create = () => {
+    storyService.createStory("me", story, image);
+  }
+
   return (
     <div className="container">
       <h1>Stories</h1>
+      <div className="d-flex mt-2">
+
+        <div className="mt-2 w-100">
+          <div className="border">
+            <textarea
+                placeholder="Share your story"
+                className="w-100 border-0"
+                onChange={(event) =>
+                    setStory(event.target.value)}/>
+          </div>
+        </div>
+
+        <div className="mt-2 align-self-center margin-left-8">
+          {
+            image ?
+                <label onClick={() => setImage(null)} className="fas fa-images fa-2x image-icon-color text-black">
+                  <i className="fa fa-remove"/>
+                </label>
+                :
+                <div>
+                  <label htmlFor="fileInput">
+                    <i className="fas fa-images fa-2x image-icon-color text-black"/>
+                  </label>
+                  <input id="fileInput" type="file"
+                         onChange={(event) =>
+                             setImage(event.target.files[0])} />
+                </div>
+          }
+        </div>
+
+        <div className="mt-2 align-self-center margin-left-8">
+          <a className="btn btn-primary btn-sm rounded-pill fa-pull-right text-white
```

Comment:
To disable this button if image is null, you can add this code `${image ? '' : 'disabled'}`

Reply:
Thank you! I have disabled the button the way you have suggested.